posthog/api/survey.py

```diff
@@ -134,6 +134,10 @@ def validate_appearance(self, value):
                 "You need to upgrade to PostHog Enterprise to use HTML in survey thank you message"
             )
 
+        survey_popup_delay_milliseconds = value.get("surveyPopupDelay")
```

Comment:
I think we accept `seconds` in the UI, so this should probably match. might be helpful to add the units to the `surveyPopupDelay` property to reduce confusion here.